docs/extensions/accounts/_index.md

```diff
@@ -17,4 +17,42 @@ oCIS needs to be able to identify users. Without a non reassignable and persiste
 
 ## Table of Contents
 
-{{< toc-tree >}}
\ No newline at end of file
+{{< toc-tree >}}
+
+
+## Adding users from the commad line.
+
+The fastest way to create a user is by executing the following command:
+
+
+    ./ocis accounts add --username johndoe --mail jondoe@none.com --password p123p123 --description  "John Doe's account" --displayname "John Doe" --enabled 
+
+For detailed information just type:
+
```

Comment:
I would vote against printing the whole help in the docs. It is subject to change.

I'd say it is enough to say:

    For detailed information just type òcis accounts add --help`. 

Also keeps the docs compact.